pkg/agent/npl/rules/rules.go

```diff
@@ -0,0 +1,31 @@
+// +build !windows
+
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rules
+
+type PodPortRules interface {
+	Init() bool
+	AddRule(port int, podip string) bool
+	DeleteRule(port int, podip string) bool
+	GetAllRules() (map[int]string, bool)
+	DeleteAllRules() bool
+}
```

Comment:
I still don't understand why these interface functions return booleans instead of errors. IMO these should return an `error` and the caller should handle the logging.